src/components/OnBoarding/ThirdForm.jsx

```diff
@@ -0,0 +1,353 @@
+        <Text paddingLeft="10vh" marginTop="5vh" fontSize="3xl" fontWeight="bold" color="#359797">
+          Become a Donation Site
+        </Text>
+        <Text paddingLeft="10vh" fontSize="sm">
+          To get started, fill out the form below.
+        </Text>
+        {termsAndConditionsIsOpened ? (
+          <Modal isOpen={true} size="3xl" isCentered>
+            <ModalOverlay>
+              <ModalContent paddingBottom="1vh" alignItems="center">
+                <ModalHeader>
+                  {' '}
+                  <Text marginLeft="1vh" fontSize="2xl">
+                    Terms and Conditions{' '}
+                  </Text>
+                </ModalHeader>
+                <ModalBody>
+                  <Text fontWeight="bold" fontSize="sm">
+                    I hereby acknowledge and agree to serve as a member/volunteer for Feeding Pets
+                    of the Homeless, 710 West Washington Street, Carson City, NV 89703. I give my
+                    consent and agree to release, indemnify and hold harmless Feeding Pets of the
+                    Homeless (DBA: Pets of the Homeless) and all personnel, including but not
+                    limited to its officers, agents and/or employees, other participants, sponsors,
+                    advertisers, I hereby assume all of the risks of participating as a volunteer,
+                    with respect to any and all injury, disability, death or loss or damage of
+                    person or property, whether arising from negligence of the releases or
+                    otherwise, to the fullest extent permitted by law. I agree to comply with the
+                    guidelines and conditions for participation. See Fundraising Policies here:
+                    http://www.petsofthehomeless.org/fundraising-policy-for-pets-of-the-homeless/
+                    Pets of the Homeless reserves the right to terminate this agreement at any time.
+                    I also grant Feeding Pets of the Homeless the right to use photographs of me at
+                    events and activities and use the photographs in future advertising including
+                    online webpage. I acknowledge that all information regarding Pets of the
+                    Homeless' operations, procedures, contacts, volunteers, recipients and donors is
+                    of a proprietary nature and should not be disclosed or used for any purposes
+                    other than the direct benefit of the organization. I acknowledge that I am not
+                    authorized to speak to the media. I agree to refer all media inquiries to
+                    Headquarters, 775-841-7463.
+                  </Text>
+                  <Text fontsize="sm" marginTop="5vh">
+                    By checking the "I Accept the Terms and Conditions" box and clicking the
+                    "Submit" button on this page, you acknowledge you have read and agree to the
```

Comment:
🚫 **[eslint]** <[react/no-unescaped-entities](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/no-unescaped-entities.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.